docs/installation.rst

```diff
@@ -0,0 +1,30 @@
+.. _installation:
+
+Installation
+============
+
+Minimum requirements
+--------------------
+
+* `cftime <https://unidata.github.io/cftime>`_ >= 1.5
+* `matplotlib <https://matplotlib.org/stable>`_
+* `numpy <https://numpy.org/doc/stable>`_
+* `python <https://docs.python.org/3>`_ >= 3.7
+
+Test requirements
+-----------------
+
+For testing, nc-time-axis also depends on pytest:
+
+* `pytest <https://docs.pytest.org/en/6.2.x>`_ >= 6.0
```

Comment:
It seems surprising to me that you would have the instructions on how to test something before you tell basic users how to install it. (I'm personally in the camp of not bothering to document installation requirements like this, and instead encode it in the ``setup.py`` (or equivalent), but that is not a universal position)

Reply:
That makes sense.  I went ahead and removed the testing requirements from the documentation in #89.  It occurred to me there are additional requirements for building the documentation as well, which I had neglected document.  I added those to the `setup.cfg` file where the testing requirements were documented.  Testing and building documentation are both things that I would mainly expect developers to be concerned with, so it is reasonable that they would only be discoverable there.

It is true that having the minimum requirements documented in multiple places opens the door for them to get out of sync, but I do think it is helpful for non-developers to see those without having to look at the source code.